libs/community/tests/integration_tests/document_loaders/docker-compose/postgresql.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+
+services:
+  postgresql:
+    image: postgres:16
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    ports:
+      - "5432:5432"
```

Comment:
Thank you! I want to move these compose files to root folder under docker so they are easy to find and are re-usable across tests that use the same services. (I can move it later in a separate PR it can stay right here for now)

Reply:
Moving this to root level docker